src/tests/nunit.runner.tests.uwp/MainPage.xaml.cs

```diff
@@ -1,4 +1,4 @@
-﻿// Copyright (c) 2015 CNUnit Project
+﻿// Copyright (c) 2018 Rob Prouse
 //
```

Comment:
Were we not going for "NUnit Project" here? I don't mind which - but the inconsistency is odd. 🙂

Reply:
I was going to switch the entire project over to my name after the discussion with Charlie about Nunit Project not being a real thing. I'll do that as a separate PR, but this error caught my attention and my OCD couldn't leave it 😄